Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1376,8 +1394,8 @@
     <RegEx find="pelova" replaceWith="rica" />
     <RegEx find="([sS])pase(\b|[mšt])" replaceWith="$1pasi$2" />
     <RegEx find="([sS])pasen([aeiou])" replaceWith="$1pašen$2" />
-    <RegEx find="spolja?š?nj?([aeiu])" replaceWith="vanjsk$1" />
-    <RegEx find="Spolja?š?nj?([aeiu])" replaceWith="Vanjsk$1" />
+    <RegEx find="spolja?š?nj?" replaceWith="vanjsk" />
```

Comment:
@xylographe how do I know if questionmark in the end is for letter or for whole word?